### PR TITLE
Fixup lints and remove github workflow

### PR DESCRIPTION
**Pull request by markan**
Fixup lints and remove github workflow
    
* Move sha256 to variable
* Remove noarch: python
* Fix doc url
    
Signed-off-by: Mark Anderson <manderson@anaconda.com>

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,24 +1,25 @@
 {% set name = "jupyter_events" %}
 {% set version = "0.6.3" %}
+{% set sha256 = "9a6e9995f75d1b7146b436ea24d696ce3a35bfa8bfe45e0c33c334c79464d0b3d" %}
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: used once only so no need to introduce a new variable

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter_events-{{ version }}.tar.gz
-  sha256: 9a6e9995f75d1b7146b436ea24d696ce3a35bfa8bfe45e0c33c334c79464d0b3
+  sha256: {{ sha256 }}
 
 build:
-  noarch: python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:13]: skip py<37, see https://github.com/jupyter/jupyter_events/blob/3f03557f824ed75f20df3d14423ab0d2f6503f40/pyproject.toml#L9

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,24 +1,25 @@
...
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   number: 0
 
 requirements:
   host:
     - pip
     - python >=3.7
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: If we build an arch-specific python package then there should be a plain `python` without pinning in `host` and `run`, see https://github.com/anaconda-distribution/perseverance-skills/blob/8a5a21c8a0167ec2dd20e8210f4b9a6968a9fad9/sections/02_Package_building/02_Recipes/Recipe_standards.md?plain=1#L83

**Review by skupr-anaconda — COMMENTED** [review]
`pip install -e ".[test]"` in test/commands is redundant. Add test dependencies to test/requires

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**